insolar/payload/payload.proto

```diff
@@ -9,11 +9,11 @@ import "github.com/insolar/insolar/insolar/record/record.proto";
 message Meta {
     uint32 Polymorph = 16;
 
-    bytes Payload    = 20;
```

Comment:
return whitespaces, pls

Reply:
@andreyromancev  is there some rules for formating this file?